src/test/lib/HandshakeTest.cpp

```diff
@@ -264,6 +276,11 @@ QuicTestConnect(
                         QUIC_LOCALHOST_FOR_AF(QuicAddrFamily),
                         ServerLocalAddr.GetPort()));
 
+                if (AsyncTicketValidation) {
+                    CxPlatSleep(1000);
```

Comment:
You need to wait until after the server has accepted the connection before you can use `Server`. See the `if (AsyncConfiguration)` scope below.